fang/README.md

```diff
@@ -4,7 +4,7 @@
 
 # Fang
 
-Background task processing library for Rust. It uses Postgres DB as a task queue.
+Background task processing library for Rust. It can use PostgreSQL, SQLite or MySQL as a task queue.
```

Comment:
I think it's only supported in async right? for blocking only postgres is supported

Reply:
yes blocking is the other PR we should specify this is for `asynk` feature

Reply:
[README stuff](https://github.com/ayrat555/fang/pull/141/commits/e7c15bb474e00e882bdff25d8bd9200549c1dffd)